src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.props.in

```diff
@@ -3,6 +3,7 @@
 		<XamarinAndroidVersion>@PACKAGE_VERSION@-@PACKAGE_VERSION_BUILD@</XamarinAndroidVersion>
 		<_JavaInteropReferences>Java.Interop;System.Runtime</_JavaInteropReferences>
 		<DependsOnSystemRuntime Condition=" '$(DependsOnSystemRuntime)' == '' ">true</DependsOnSystemRuntime>
+		<ImplicitlyExpandNETStandardFacades>false</ImplicitlyExpandNETStandardFacades>
```

Comment:
Should this be overridable?

Reply:
It is a common (maybe "de facto" standard) MSBuild property to control whether to include those references or not.

Reply:
the iOS/Mac tooling does not allow this to be overridable. I suspect this is by design so NetStandard Facades are never included in the output.